### PR TITLE
Added r-JustifyAlpha as an R pacakge

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge/help-r First time trying to contribute an R package, pointers welcome!

upstream: https://cran.r-project.org/web/packages/JustifyAlpha/index.html

**Comment by thewchan** [comment]
@conda-forge/staged-recipes All tests past, but this is my first R package just wanted to make sure I did everything correct. Thanks!

File: recipes/r-justifyalpha/build.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# 'Autobrew' is being used by more and more packages these days
```
Comment (cbrueffer) [inline recipes/r-justifyalpha/build.sh:3]: Please remove all comments in this file (minus the #! line of course).

File: recipes/r-justifyalpha/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: knitr, rmarkdown, testthat,
```
Comment (cbrueffer) [inline recipes/r-justifyalpha/meta.yaml:24]: This can be removed.

File: recipes/r-justifyalpha/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    paradox, or by minimizing or balancing error rates. For more information about the
+    package please read the following: Maier & Lakens (2021) <doi:10.31234/osf.io/ts4r6>).'
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - thewchan
```
Comment (cbrueffer) [inline recipes/r-justifyalpha/meta.yaml:70]: Please add `conda-forge/r` as additional maintainer.

**Review by cbrueffer — COMMENTED** [review]
Looks fine, just a few comments to clean things up.

File: recipes/r-justifyalpha/build.sh (hunk 1 above)
Comment (thewchan) [reply recipes/r-justifyalpha/build.sh:3]: ✔️

File: recipes/r-justifyalpha/meta.yaml (hunk 2 above)
Comment (thewchan) [reply recipes/r-justifyalpha/meta.yaml:24]: Just the comment right? Still need `lib/R/lib` and `lib/`?

File: recipes/r-justifyalpha/meta.yaml (hunk 3 above)
Comment (thewchan) [reply recipes/r-justifyalpha/meta.yaml:70]: ✔️

**Review by cbrueffer — APPROVED** [review]

File: recipes/r-justifyalpha/meta.yaml (hunk 2 above)
Comment (thewchan) [reply recipes/r-justifyalpha/meta.yaml]: ✔️

**Comment by thewchan** [comment]
@conda-forge/staged-recipes @cbrueffer this is now ready for final review!

**Pull request merged**